src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -1528,6 +1534,11 @@ private static void SafeAssert(bool condition, string message, [CallerMemberName
             }
         }
 
+        ValueTask IAsyncDisposable.DisposeAsync()
```

Comment:
We need to keep the Task returning DisposeAsync to avoid making a breaking change

Reply:
😭